views/default/core/js/upgrader.js

```diff
@@ -158,7 +152,7 @@ define(['jquery', 'elgg/Ajax', 'elgg/spinner', 'elgg/popup', 'elgg/system_messag
 					fakeDone = Math.round(FAKE_TOTAL - (FAKE_TOTAL - fakeDone) / 2);
 					progressbar.progressbar({value: fakeDone});
 				} else {
-					percentage = parseInt(numProcessed * 100 / total);
+					percentage = numProcessed * 100 / total;
```

Comment:
did we not use parseInt here to round floats to int?

Reply:
updated